tests/test_eigen.h

```diff
@@ -63,4 +63,50 @@ PrintTo( Vector2d const& v, ::std::ostream* os )
 
 } // end namespace Eigen
 
+// ----------------------------------------------------------------------------
+template <typename A, typename B>
+bool
+CompareMatrices( A const& m1, B const& m2 )
```

Comment:
KWIVER style generally uses `lower_case_with_underscores`, or are you trying to match the coding style of Eigen?

Reply:
Actually, I was trying to follow Google Test style. However, I figured out a better way to write these helpers (using a functor for overload resolution instead of `decltype` trickery), and have also moved them into the `kwiver::testing` namespace, and they are now following KWIVER naming convention.